### ISSUE TITLE
deepseq-1.4.6.0 is not on Hackage

### ISSUE THREAD

**Issue by phadej** [haskell/deepseq#66]
but already bundled with GHC-9.2.0-rc1.

(Also there is no changelog entry in the repository, or even a tag, which is what I was looking for).

**Comment by chessai** [comment]
What is the commit GHC is using as 1.4.6.0? (I don't have time to look right now). Does it just include the change for removing Option instances as compared to 1.4.5.0? cc @bgamari

**Comment by phadej** [comment]
It might make sense to wait what happens with `Solo` in `base` and resolve #67 before making proper release.

**Comment by phadej** [comment]
GHC-9.2.1 is released, `deepseq-1.4.6.0` is not on Hackage yet.

**Comment by phadej** [comment]
GHC-9.2.1 is released, `deepseq-1.4.6.0` is not on Hackage yet.

cc @chessai, @Bodigrim, @bgamari

**Comment by phadej** [comment]
Also:

```
% ghci-9.2.1
GHCi, version 9.2.1: https://www.haskell.org/ghc/  :? for help
Loaded GHCi configuration from /home/phadej/.ghci
Prelude> :m +Data.Tuple
Prelude Data.Tuple> :m +Control.DeepSeq 
Prelude Data.Tuple Control.DeepSeq> force (Solo 'x')

<interactive>:3:1: error:
    • No instance for (NFData (Solo Char))
```

So #67 didn't make it into the `1.4.6.0`. I'm slightly disappointed.

**Comment by Bodigrim** [comment]
@phadej I'm not a maintainer of `deepseq`.

**Comment by phadej** [comment]
@Bodigrim you are CLC chair, and `deepseq` is an owner of `deepseq`. Maintainer or `deepseq`, herbert, is absent.

**Comment by Bodigrim** [comment]
https://hackage.haskell.org/package/deepseq/maintainers/ lists @chessai as a maintainer, but I don't know whether it was a voluntary choice on his part or just temporarily substituting for Herbert as a CLC Chair. If it's the latter (@chessai could you please clarify?), we should source and install a new maintainer indeed.

**Comment by chessai** [comment]
> https://hackage.haskell.org/package/deepseq/maintainers/ lists @chessai as a maintainer, but I don't know whether it was a voluntary choice on his part or just one-off substituting for Herbert. If it's the latter (@chessai could you please clarify?), we should source and install a new maintainer indeed.

I am listed because of the lack of maintainers hip. I have helped with deepseq a bit in the last few months, but am far from a maintainer of it. We need to source a new maintainer(s).

**Comment by cigsender** [comment]
I'd be willing to take over maintainership of this package for the time being, to help push issues through that have been stagnating.

**Comment by Bodigrim** [comment]
@cigsender great!

Dear CLC members, I suggest we approve installing @cigsender as a new maintainer of `deepseq`. Please vote below. 
@chessai @emilypi @cgibbard @gwils 

+1.

**Comment by cgibbard** [comment]
+1

**Comment by emilypi** [comment]
+1

**Comment by Bodigrim** [comment]
Assuming @cigsender votes for herself, we have 4 votes in favor out of 6. Assuming she abstains because of the conflict of interests, we have 3 votes in favor out of 5. Both ways the suggestion is approved. 

@emilypi @chessai could you please grant @cigsender GitHub maintainer bits for this repo? 

@gbaz could you please give @cigsender Hackage maintainers rights for `deepseq`? @cigsender what is your Hackage username?

**Comment by cigsender** [comment]
@Bodigrim my hackage username is `melaniebrown`.

**Comment by chessai** [comment]
> Assuming @cigsender votes for herself, we have 4 votes in favor out of 6. Assuming she abstains because of the conflict of interests, we have 3 votes in favor out of 5. Both ways the suggestion is approved.
> 
> @emilypi @chessai could you please grant @cigsender GitHub maintainer bits for this repo?
> 
> @gbaz could you please give @cigsender Hackage maintainers rights for `deepseq`? @cigsender what is your Hackage username?

@Bodigrim and @cigsender you should have received invites to the CLC Haskell GitHub team. This will grant you access to all core libraries.

**Comment by gbaz** [comment]
> @gbaz could you please give @cigsender Hackage maintainers rights for deepseq? @cigsender what is your Hackage username?

done. Note that because chessai is in the maintainers group, that could have been handled directly :-)

**Comment by Bodigrim** [comment]
Cool. We need to figure out which commit was shipped with GHC 9.2.1 and make a matching release.

**Comment by cigsender** [comment]
https://hackage.haskell.org/package/deepseq-1.4.6.0
